test/mock_data.py

```diff
@@ -16,6 +16,76 @@
 # at runtime
 CREATED_MODIFIED_GET_DATA_EXPECTED = {"created_time": ANY, "modified_time": ANY}
 
+# --------------------------------- MANUFACTURERS ---------------------------------
```

Comment:
I slightly changed this in https://github.com/ral-facilities/inventory-management-system-api/pull/315, if you are happy with the change it could be done here just for manufactures so it merges nicely.

Reply:
Sorry but I can't workout what I need to change here, could you please clarify?

Reply:
They are all in the order _IN, _POST, _GET, and have a comment above on to separate each different type e.g. thee would be a comment like `# Required values only` or `# All values`. Its easiest to see on https://github.com/ral-facilities/inventory-management-system-api/blob/c3c20dbb33b73e164bb0f2751e15457062a7b8dd/test/mock_data.py#L231-L296. I don't think there would need be a comment above the _A and _B  manufactuerer ones though as there isn't anything special to say about them whereas systems used `# No parent`.